problem_two.rb

```diff
@@ -0,0 +1,10 @@
+# Given an integer array of size N, identify the largest 2 numbers in this array. Do not sort the array.
+
+class ProblemTwo
+  def max_two(array)
+    raise 'Invalid input: array must contain at least two numbers' \
+      if array.length < 2
```

Comment:
There's no assumption in the problem statement that the array MUST contain at least two numbers.